daprdocs/content/en/operations/components/setup-bindings/supported-bindings/mysql.md

```diff
@@ -0,0 +1,148 @@
+---
+type: docs 
+title: "MySQL binding spec"
+linkTitle: "MySQL"
+description: "Detailed documentation on the MySQL binding component"
+---
+
+## Setup Dapr component
+
+```yaml
+apiVersion: dapr.io/v1alpha1
+kind: Component
+metadata:
+  name: <NAME>
+  namespace: <NAMESPACE>
+spec:
+  type: bindings.mysql
+  version: v1
+  metadata:
+    - name: url # Required, define DB connection in DSN format
+      value: <CONNECTION_STRING>
+    
```

Comment:
There's a space here. is something missing maybe?

Reply:
the blank line is removed.